cmd/cmdtest/policy/commands.ct

```diff
@@ -28,3 +28,6 @@ EOF
 sink<<EOF
 ["$ECR/tyk-sink:master"]
 EOF
+exclude<<EOF
+[]
```

Comment:
The code in defaults below has exclusions, so how come this array is empty?

Reply:
$ setenv IS_PR yes is not default given the case is within is_pr trigger:

<img width="789" alt="image" src="https://github.com/TykTechnologies/gromit/assets/35462288/7e410d7f-1e19-43ce-bf40-946e0b3bb0dc">